ddtrace/appsec/_iast/taint_sinks/header_injection.py

```diff
@@ -47,8 +47,12 @@ def _(m):
     @when_imported("django.http.response")
     def _(m):
         trace_utils.wrap(m, "HttpResponse.__setitem__", _iast_h)
-        trace_utils.wrap(m, "ResponseHeaders.__setitem__", _iast_h)
         trace_utils.wrap(m, "HttpResponseBase.__setitem__", _iast_h)
+        try:
+            trace_utils.wrap(m, "ResponseHeaders.__setitem__", _iast_h)
+        except AttributeError:
+            # no ResponseHeaders in django<3
+            pass
```

Comment:
I think we should do this block in a different PR and backport it to 2.7, 2.8, 2.9...

Reply:
I can do that.

Reply:
`trace_utils.wrap(m, "ResponseHeaders.__setitem__", _iast_h)` isn't in any release

Reply:
Oh yes, it was merged only 3 weeks ago. OK, I let it in that PR then, thanks @avara1986